content/200-concepts/150-data-platform/01-about-platform/100-platform-limits-and-status.mdx

```diff
@@ -0,0 +1,84 @@
+| Maximum number of connection strings per environment                                                                                                                  | 100            |
+| Response timeout in Data Browser                                                                                                                                      | 60s            |
+| Response timeout in Query Console                                                                                                                                     | 60s            |
+| Maximum number of rows fetched in Data Browser or Query Console                                                                                                       | Not applicable |
+
+## Concurrency limits of the Prisma Data Proxy
+
+The concurrency limits of the Prisma Data Proxy define the maximum number of proxy instances that can run simultaneously for a single project. By default, only one proxy instance is started for a single project environment.
+
+More proxy instances start when:
+
+- you create and start using a new environment
+- the number of running proxy instances cannot handle the number of incoming queries
+- multiple Prisma Clients connect that use a different version of the Prisma Client
+- multiple Prisma Clients connect that use a different version of the Prisma schema
+
+| Concurrency limits                                                     | Number |
+| ---------------------------------------------------------------------- | ------ |
+| Maximum number of concurrent proxy instances on **Early Adopter** plan | 3      |
+| Maximum number of concurrent proxy instances on **Free** plan          | 100    |
+| Maximum number of concurrent proxy instances on **Scale** plan         | 100    |
+| Maximum number of concurrent proxy instances on **Enterprise** plan    | Custom |
+
+A single proxy instance can handle ~1200 RPS which translates to ~69 concurrent database connections.
+
+<Admonition>
+
+**Extending the concurrency limits**<br /><br />
+Contact Support if you have a project running on the **Scale** plan and the project exceeds the current limit of the concurrent database connections.
+
+</Admonition>
+
+## Compatibility with Prisma ORM
+
+We are dedicated to keeping the Prisma Data Platform compatible with the latest version of the Prisma ORM.
+
+- **Compatibility of the platform Data Browser, Query Console, Schema with the Prisma ORM**
+  - The interfaces are updated after each Prisma ORM release and aim to support the latest version.
+  - In rare occasions, there might be several days worth of a delay before the interfaces begin to support the latest version of the Prisma ORM.
+- **Compatibility of the Prisma Data Proxy with the Prisma ORM**
+  - Prisma Data Proxy is updated almost immediately to the latest version of the Prisma ORM.
+  - Prisma Data Proxy does not support the preview features `interactiveTransactions` and `metrics`.
```

Comment:
We know that it will eventually ( iTX in next Q, metrics probably after ), so perhaps we could allude to the fact that this is coming. Otherwise it may read as if it will never support those features.

Reply:
I've slightly edited this to also say "yet". It's better to not promise future releases because things change and it then makes an even poorer impression.